lib/katello/tasks/pulp3_post_migration_check.rake

```diff
@@ -0,0 +1,30 @@
+require File.expand_path("../engine", File.dirname(__FILE__))
+
+namespace :katello do
+  desc "Runs a post Pulp3 migration check for supported content types."
+  task :pulp3_post_migration_check => :environment do
+    repository_types = Katello::Pulp3::Migration::REPOSITORY_TYPES
+
+    repository_types.each do |type|
+      repositories = Katello::Repository.with_type(type)
+        .where('version_href is NULL OR remote_href is NULL OR publication_href is NULL')
```

Comment:
Docker repositories will never have a non-nil `publication_href`.  We need to skip that check.